.github/workflows/release.yml

```
@@ -0,0 +1,41 @@
+name: Release
+
+on:
+    release:
+        types: [published]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
```

```suggestion
        runs-on: ubuntu-latest

        permissions:
            contents: write
            pull-requests: read
```
should fix the permission issue without changing the defaults for the repo

---

Nice!